apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/SwiftApi.java

```diff
@@ -17,6 +17,7 @@
 package org.jclouds.openstack.swift.v1;
 
 import java.io.Closeable;
+
```

Comment:
Please sweep through the entire pull request and remove these whitespace changes.